Server/mods/deathmatch/logic/CConsoleCommands.cpp

```diff
@@ -1827,6 +1827,17 @@ bool CConsoleCommands::AuthorizeSerial( CConsole* pConsole, const char* szArgume
     return false;
 }
 
+bool CConsoleCommands::ReloadAcl(CConsole* pConsole, const char* szArguments, CClient* pClient, CClient* pEchoClient)
+{

```

Comment:
Needs to make sure only console / people with rights can use this command. [See this example.](https://github.com/Timic3/mtasa-blue/blob/7bf2d93793053bbdf8927ed1fb37ee1c6a161d83/Server/mods/deathmatch/logic/CConsoleCommands.cpp#L1699)